### PR TITLE
Create README-zh.md

### PR DESCRIPTION
**Pull request by zhangsanlzh**
Hi, I just think the project is very useful and interested in it. Therefore, i try to translate the README file to Chinese. 

Hope you can accept it.Best Wishes!

### CODE REVIEW

**Comment by zhangsanlzh** [comment]
ok，i will ☺️

**Comment by aalhour** [comment]
Hi @zhangsanlzh, please rebase your branch against upstream/master. Your PR is trying to push an outdated version of `README.md`

File: README.md [hunk 1]
```diff
@@ -1,147 +1,210 @@
-# C# ALGORITHMS [![Build Status](https://travis-ci.org/aalhour/C-Sharp-Algorithms.svg?branch=master)](https://travis-ci.org/aalhour/C-Sharp-Algorithms)
+```
```
Comment (aalhour) [inline README.md]: This file shouldn't be changed in your PR.

Comment (zhangsanlzh) [reply README.md]: ohhhh，i give it up😂

**Pull request closed without merging**